src/wx/panel.cpp

```diff
@@ -461,6 +461,10 @@ void GameArea::recompute_dirs()
         batdir = wxGetApp().GetAbsolutePath(gopts.battery_dir);
     }
 
+    if (!wxIsWritable(batdir)) {
+        batdir = wxGetApp().GetAbsolutePath(get_xdg_user_data_home() + DOT_DIR);
```

Comment:
Can we make a convenience method on the app for this, e.g. `GetConfigDir()` or something like that.